### PR TITLE
-System-ui info updated

### PR DESCRIPTION
**Pull request by awebdeveloper**
1. Chrome for android supports it 

Chrome for android supports it since 56  …
https://www.chromestatus.com/feature/5640395337760768
```
Enabled by default (tracking bug) in:

Chrome for desktop release 56
Chrome for Android release 56
Android WebView release 56
Opera release 43
Opera for Android release 43
```
2, Chrome/Safari on mac also supports BlinkMacSystemFont
https://booking.design/implementing-system-fonts-on-booking-com-a-lesson-learned-bdc984df627f

### CODE REVIEW

File: features-json/font-family-system-ui.json [hunk 1]
```diff
@@ -150,9 +150,9 @@
       "50":"n",
       "51":"n",
       "52":"n",
-      "53":"n",
-      "54":"n",
-      "55":"n",
+      "53":"n#2",
+      "54":"n#2",
+      "55":"n#2",
```
Comment (Fyrd) [inline features-json/font-family-system-ui.json]: Need a space between `n` & `#2` here

**Comment by awebdeveloper** [comment]
@Fyrd done

File: features-json/font-family-system-ui.json [hunk 2]
```diff
@@ -303,9 +303,10 @@
   },
   "notes":"",
   "notes_by_num":{
-    "1":"Supported as the `-apple-ui` value (only on Mac)"
+    "1":"Supported as the `-apple-ui` value (only on Mac)",
+    "2":"Supported as the 'BlinkMacSystemFont'  value (only on Mac)",
```
Comment (cvrebert) [inline features-json/font-family-system-ui.json]: Delete trailing comma (it's not legal in JSON).

Comment (Fyrd) [reply features-json/font-family-system-ui.json]: @awebdeveloper Can you fix this? Thanks.

**Comment by awebdeveloper** [comment]
@cvrebert @Fyrd done

**Comment by Fyrd** [comment]
Thanks!

**Pull request merged**